character-gallery/src/CharacterList.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import CharacterItem from './CharacterItem.js';
+import {Link} from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+export default class CharacterList extends Component {
+    render() {
+        const characters = this.props.characterList.map((character, idx) => {
+        return (
+            <Link className="char-url" to={character.name}><CharacterItem key={idx} characterItem={character.name}/></Link>

```

Comment:
Believe you need to include the "/" before character.name for proper routing. 
`to={`/${character.name}`}`